common/acknowledgements/aria-wg-active.html

```diff
@@ -95,5 +95,6 @@ <h4>Participants active in the ARIA WG at the time of publication</h4>
 		<li>Benjamin Young (Wiley)</li>
 		<li>Valerie Young (Bocoup)</li>
 		<li>Marco Zehe (Mozilla Foundation)</li>
+		<li>Keith Cirkel (GitHub)</li>
```

Comment:
hi, sorry that it is not clear -- but for (1) the "common" folder is automatically pulled in from a different repo, and for (2) we actually now auto generate the aria working group members and participants section (you will be added based on your participation in github), so these files are defunct, I'll make an issue to remove them :)

So please remove this from the diff?